bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/Sleeper.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc.async;
+
+/**
+ * An interface that wraps Thread.sleep for test purposes 
+ */
+public interface Sleeper {
+  void sleep(long ms) throws InterruptedException;
+
+  static Sleeper THREAD_SLEEPER = new Sleeper() {
```

Comment:
Since we want multiple implementations for test purposes, INSTANCE seems less informative than THREAD_SLEEPER.  Maybe DEFAULT_INSTANCE.